app/services/outage_status.rb

```diff
@@ -49,6 +56,24 @@ def phone_finder_outage?
     all_vendor_outage?([:lexisnexis_phone_finder])
   end
 
+  def idv_scheduled_maintenance_status
+    if idv_scheduled_maintenance?
+      :full_outage
+    else
+      :operational
+    end
+  end
+
+  # @return [Boolean] returns true when we are currently within a scheduled maintenance window
+  def idv_scheduled_maintenance?(now: Time.zone.now)
+    start = IdentityConfig.store.vendor_status_idv_scheduled_maintenance_start
+    finish = IdentityConfig.store.vendor_status_idv_scheduled_maintenance_finish
+
+    if start.present? && finish.present?
+      (start.in_time_zone('UTC')...finish.in_time_zone('UTC')).cover?(now)
```

Comment:
I have concerns about `.in_time_zone('UTC')`. Could you add a spec that sets up the start and finish configs in a different timezone and make sure the conversion includes/excludes a time the way you expect?

Also I wonder if you need a try/catch here in case someone puts in an invalid date in the start/finish configs.

Reply:
`String#in_time_zone` is added by Rails and is very forgiving with format, it will return `nil` if it can't parse

The purpose of it is to allow us to author the configs in different timezones, so say if we author this upcoming maintenance window in eastern timezone:
```ruby
"2023-11-12T07:00:00-05:00".in_time_zone("UTC")
=> Sun, 12 Nov 2023 12:00:00.000000000 UTC +00:00
```